Sources/Gui/Main.cpp

```diff
@@ -57,13 +57,23 @@
 #endif
 
 
+#ifdef _MSC_VER>=1900	// Visual Studio 2015 or higher
+extern "C" { FILE __iob_func[3] = { *stdin,*stdout,*stderr }; }
```

Comment:
> But it turns out if I include the 2 images at compile time (with #include like it was) I do get errors inside VS but the project actually builds... 

Intellisense's problem. Let's ignore it for now.

>  When I run it I get this error (SystemMessages.log is empty) Why could it fail to build the scripts?.

I don't know. I remember that someone has reported a similar issue, but couldn't solve it because of the lack of information. Setting a breakpoint [here](https://github.com/yvt/openspades/blob/master/Sources/ScriptBindings/ScriptManager.cpp#L45) and inspecting the error message might help.